alvr/xtask/src/dependencies.rs

```diff
@@ -11,6 +11,24 @@ pub fn choco_install(sh: &Shell, packages: &[&str]) -> Result<(), xshell::Error>
     .run()
 }
 
+pub fn prepare_openvr_sdk() {
+    let sh = Shell::new().unwrap();
+
+    let deps_path = afs::deps_dir().join("openvr");
+    sh.remove_path(&deps_path).ok();
+
+    command::download_and_extract_zip(
+        &format!(
+            "https://github.com/ValveSoftware/openvr/archive/refs/tags/{}",
+            "v1.16.8.zip"
```

Comment:
Extract "1.16.8" version string to a separate constant (defined inside the function) for easier upgrade.